tendrl/integrations/gluster/objects/definition/gluster.yaml

```diff
@@ -1,26 +1,22 @@
 ---
 namespace.integrations.gluster:
-  flows:
```

Comment:
Accidentally removed?

Reply:
@shtripat told this is just sample and this cause issue when i run code so he suggested me to remove it

Reply:
Ok I thought it was accidentally removed